### DISCUSSION TITLE
More than a click away, Android devices drag and click actions

### DISCUSSION THREAD

**Discussion by RocketSheepOriginal** [ppy/osu#15193] [Ideas] [answered]
I discovered recently that osu for androids was already available to play, when I gave it a shot I was confused that the game only recognizes one single tap, which is different from the unofficials games _osu!droid_ and _opsu!_.

My idea is that it could have a **mod** or any other **configuration** that you can use the first finger recognized to drag the cursor and the other recognized fingers being *clicks* like a key.
For that mod you couldn't just tap and the cursor **moves and tap**, the tap should only moves the cursor to that position, maybe it could have buttons on the screen where you need to click in specific to count as a valid tap. (You could select where this buttons are placed in more detailed configurations).

I personally don't know if that is recognized as a _playstyle_ for android / mobile devices, but I think it could be a useful mod if can't be used as a playstyle. (And I don't know to use github, sorry)

**Comment by integerrr** [comment, marked as answer]
#4201
